src/contact.rs

```diff
@@ -498,6 +512,11 @@ impl Contact {
         Ok(())
     }
 
+    /// Returns whether contact is a bot.
+    pub fn is_bot(&self) -> bool {
```

Comment:
@Septias hi, this was added but in the end is not public via C bindings nor JSONRPC API, so it is not possible to use other than using Rust directly, 

now that the address is hidden in the chat for verified contacts, I was considering to display "bot" label there for bots, to give some use the the new free space since without address it looks ugly, by the way this is what telegram does

cc @link2xt, @iequidoo, @r10s

Reply:
Made a pr: #5197